conans/client/cmake.py

```diff
@@ -156,3 +165,48 @@ def runtime(self):
         if runtime:
             return "-DCONAN_LINK_RUNTIME=/%s" % runtime
         return ""
+
+    def configure(self, conan_file, args=None, vars=None, source_dir=None, build_dir=None):
```

Comment:
Hi! "vars" are not C preprocessor defines but CMake variables so it might be misleading to name them "defs", however I agree "vars" is not super clear either. Naming is hard...